chart/prometheus-msteams/templates/configMapTemplate.yaml

```diff
@@ -10,3 +10,11 @@ binaryData:
 {{- else }}
```

Comment:
How about completely removing `.Values.customCardTemplate` in favor of `.Values.customCardTemplates`?

Reply:
If it's okay to make a non backward compatible change here that's fine for me. So we'd go to version `0.8.0` then?

Reply:
If it's okay to make a non backward compatible change here that's fine for me. So we'd go to version `0.8.0` then?

Reply:
Yeah why not, but as of my comments below, we would also remove `.Values.customCardTemplates`. As this is a breaking change, it would be rather version `1.0.0`

Reply:
Changed version to `1.0.0`